zeebe/backup/pom.xml

```diff
@@ -70,6 +75,18 @@
       <artifactId>simpleclient</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-atomix-cluster</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.agrona</groupId>
+      <artifactId>agrona</artifactId>
+      <scope>provided</scope>
```

Comment:
:wrench: I assume this was automatically added? Not sure it makes sense to have provided here.

Reply:
Unforunately it's really used in "source" code only in the test scope. 

However, it's also used in the compile scope by the journal module, so If I set `scope=test` then the compile task does not succeed. 

However If I don't put the `scope=test` the lint task does not succeed because it should be put with `scope=test`.

I hope I explained the situation, is there a better solution?

Reply:
Ah, maven :sweat_smile: 

No, it was just unexpected to have `provided` here, that's all. That said, can we set it to test if we remove the dependency on the `journal` module?

Reply:
Yes probably, but we cannot depend on `zeebe-atomix-cluster` as well. Is there a way to ignore this error when linting? this is more of a linter issue rather than maven per se.

Reply:
Fixes by configuring the plugin to ignore false positives